packages/side-nav/src/vaadin-side-nav-base-styles.js

```diff
@@ -6,97 +6,101 @@
 import { css } from 'lit';
 
 export const sideNavItemBaseStyles = css`
-  :host {
-    display: block;
-  }
+  @layer base {
```

Comment:
This file should not be included to this PR as it's not related to base styles. Let's exclude it from this PR.
We'll probably need to rename it to `core-styles.js` but let's do that in a separate PR.